### PR TITLE
Ocrvs 5794 - configurable signature for different users

### PR DESCRIPTION
**Pull request by Nil20**
Farajaland PR: https://github.com/opencrvs/opencrvs-farajaland/pull/754

### CODE REVIEW

**Comment by euanmillar** [comment]
@Nil20 as this is a hotfix PR, you need to add the Hotfix label please

File: packages/config/src/handlers/application/applicationConfigHandler.ts [hunk 1]
```diff
@@ -22,6 +22,16 @@ import { getDocumentUrl } from '@config/services/documents'
...
 
+export const SystemRoleType = [
+  'FIELD_AGENT',
+  'LOCAL_REGISTRAR',
+  'LOCAL_SYSTEM_ADMIN',
+  'NATIONAL_REGISTRAR',
+  'NATIONAL_SYSTEM_ADMIN',
+  'PERFORMANCE_MANAGEMENT',
+  'REGISTRATION_AGENT'
+]
```
Comment (Zangetsu101) [inline packages/config/src/handlers/application/applicationConfigHandler.ts:33]: Well the sys admin & performance management roles don't really have anything to do with declarations so I guess we don't need them here?

File: packages/client/src/tests/mock-offline-data.ts [hunk 2]
```diff
@@ -478,7 +478,8 @@ export const mockOfflineData = {
     INFORMANT_SIGNATURE: false,
     INFORMANT_SIGNATURE_REQUIRED: false,
     USER_NOTIFICATION_DELIVERY_METHOD: 'email',
-    INFORMANT_NOTIFICATION_DELIVERY_METHOD: 'email'
+    INFORMANT_NOTIFICATION_DELIVERY_METHOD: 'email',
+    SIGNATURE_REQUIRED_FOR_ROLES: ['LOCAL_REGISTRAR', 'NATIONAL_REGISTRAR']
```
Comment (rikukissa) [inline packages/client/src/tests/mock-offline-data.ts:482]: @Zangetsu101 just a note for the future but maybe if would be best to start adding user features like this to `User.scope` so for instance now we'd add `scope: ['declare', 'has-signature']` when we create the users. That way, over time, we'd reduce the significance of what a "LOCAL_REGISTRAR" for instance means and countries could basically create any user type they want

**Review by rikukissa — COMMENTED** [review]
@Nil20 Looks good! Question: let's say I add a signature for field agents. How do I read the signature as part of the certificate printing?

**Comment by Nil20** [comment]
> @Nil20 Looks good! Question: let's say I add a signature for field agents. How do I read the signature as part of the certificate printing?

**Comment by Nil20** [comment]
> @Nil20 Looks good! Question: let's say I add a signature for field agents. How do I read the signature as part of the certificate printing?

https://github.com/opencrvs/opencrvs-farajaland/pull/754
@rikukissa does this help?

**Comment by rikukissa** [comment]
> > @Nil20 Looks good! Question: let's say I add a signature for field agents. How do I read the signature as part of the certificate printing?
> 
> [opencrvs/opencrvs-farajaland#754](https://github.com/opencrvs/opencrvs-farajaland/pull/754) @rikukissa does this help?

Thanks, it does! Left a couple of comments there

**Pull request merged**